ui-pf4/src/main/webapp/src/layout/TransformationPaths.ts

```diff
@@ -119,10 +119,15 @@ export const AZURE: TransformationPathOption = {
   description:
     "Upgrade your Java application so it can be deployed in different flavors of Azure.",
   options: [
+    {
+        label: "azure-spring-apps",
+        value: "azure-spring-apps",
+        default: true,
+      },
     {
       label: "azure-appservice",
       value: "azure-appservice",
-      default: true,
+      default: false,
```

Comment:
@showpune has the default value being changed on purpose from `azure-appservice` to `azure-spring-apps`?Because it's a kind of "breaking change" for the Windup Web Console users that are used to have `azure-appservice` as the default value for the `Azure` card in the Migration Paths page.

Reply:
Since we have a deadline for an upcoming release, the Windup team decided to take the most conservative approach with this change, pushing directly to this PR the c8d938b commit:

- `azure-appservice` value is still the default target in the `Azure` target
- `azure-spring-apps` value has been added at the end

Screenshot reflects what described
![Screenshot from 2023-04-12 15-32-21](https://user-images.githubusercontent.com/7288588/231478272-61973599-d619-4299-b5e8-b34b84010e55.png)

Reply:
sure!